### ISSUE TITLE
Is this repository managed now?

### ISSUE THREAD

**Issue by TyeolRik** [ceph/ceph-rust#86]
Hello, I am newbie on the language rust.

I am searching for some languages which should I use. Our team is now using C++ for Ceph controller now but, we are trying to find a new area. (and also solving some memory issues.) (I already watched that go-ceph is maintained well now.)

I can see that the last commit was 2 years ago on your repository master branch. (Sadly, too old to be support recent ceph version like 18, reef)

Is this repository managed now? (Or can this repository support 18.2.0 version now?)

**Comment by cholcombe973** [comment]
I don't think redhat is maintaining this repo any more. It's prob worth forking it.

**Comment by Xuanwo** [comment]
Yes, for now, let's go! I just received permission to work on this project, and I'm going to revive it. Welcome to the development team!

And @cholcombe973, would you like to add me as the co-owner of `ceph` crate?
